src/context/db.tsx

```diff
@@ -8,7 +8,7 @@ import { Database } from "../types";
 
 interface DbContext {
   db?: Database;
-  setDb?: (db: Database) => void;
+  setDb?: React.Dispatch<React.SetStateAction<Database | undefined>>;
```

Comment:
We typically use the specific modules directly, i.e. `Dispatch<SetStateAction...`

Reply:
Okay got it

Reply:
Import from react above 👍